tests/test_console.py

```diff
@@ -84,7 +84,7 @@ def run_test(self):
             try:
                 subprocess.check_call(
                     [
-                        "which", sys.argv[0]
+                        "where" if os.name == "nt" else "which", sys.argv[0]
```

Comment:
I'm wondering if doing something like:
```python
import os
import sysconfig

console_script = os.path.join(sysconfig.get_path('scripts'), sys.argv[0])
if not os.path.exists(console_script):
    self.fail(
        "Could not find '{}'.  Tests that explicitly shell"
        " out can be disabled by setting the environment variable "
        "OTIO_DISABLE_SHELLOUT_TESTS.".format(console_script)
    )
```
might be a little bit better. It would at least guarantee that the console script is really the one we are looking for.

Reply:
it's true that would be a more exact test

Reply:
Although in common use, "which" is not POSIX, and is being deprecated by Debian. "command -v" is the POSIX compliant alternative. https://lwn.net/ml/debian-devel/9e3a2df146f73437fe8cce10272ddf3913e48757.camel@debian.org/

Reply:
Removing `which` is weird, it's been in common use for decades. I took a peek at the issue thread, sigh... If some yak shaving is going on in the debian bike shed, then using Jean-Christophe's suggestion is a good way to avoid having a yak in that race.

Reply:
Unfortunately this fails on Windows as the files all have a `.exe` extension. Would this be an exceptable addition:
```python
 console_script = os.path.join(sysconfig.get_path('scripts'), sys.argv[0])
            if not os.path.exists(console_script):
                # We might be in Windows
                if not os.path.exists(console_script + ".exe"):
                    self.fail(
                    "Could not find '{}'.  Tests that explicitly shell"
                    " out can be disabled by setting the environment variable "
                    "OTIO_DISABLE_SHELLOUT_TESTS.".format(console_script)
                    )
```

Reply:
This PR was previously approved, but conversation has stalled. Where do we stand? As is? Or do we want further modifications?

Reply:
What about

```
console_script = os.path.join(sysconfig.get_path('scripts'), sys.argv[0])
If platform.platform() == 'windows':
    console_script += '.exe'

if not os.path.exists(console_script):
    self.fail(
        "Could not find '{}'.  Tests that explicitly shell"
        " out can be disabled by setting the environment variable "
        "OTIO_DISABLE_SHELLOUT_TESTS.".format(console_script)
    )
```

Reply:
Ok, yes, that *almost* works for me. It needs to be platform.system() and check for Capitalized `Windows`. platform.platform() returns `Windows-10-10.0.19041-SP0` or similar depending... platform.system() returns `Windows`. On Mac, it platform.platform returns `macOS-12.0-arm64-arm-64bit`, platform.system() returns `Darwin`, so clearly platform() returns a tuple cross on any platform.

If you add your variant, with the change `platform.system() == 'Windows'` on the PR, I'm happy to land it when the tests pass.